packages/cdk/lib/roles/policies/toil-batch-policy.ts

```diff
@@ -0,0 +1,30 @@
+import { PolicyStatement, Effect } from "aws-cdk-lib/aws-iam";
+import { CromwellBatchPolicy } from "./cromwell-batch-policy";
+
+export interface ToilBatchPolicyProps {
+  jobQueueArn: string;
+  toilJobArnPattern: string;
+}
+
+export class ToilBatchPolicy extends CromwellBatchPolicy {
```

Comment:
Noting we should generalize this further - e.g. a ServerBatchPolicy that both Cromwell and Toil extend.